code/__DEFINES/power/power.dm

```diff
@@ -1,3 +1,7 @@
+#define kW *1000
+#define MW kW *1000
+#define GW MW *1000
```

Comment:
I wonder if we should change these not be shortened. I mean, not `kW`. to `KILOWATT`?? Not sure.
This is not blocking review. Feel free to discuss about this. (and prolly not the point of this PR)

Reply:
done